tests/tck/features/mutate/InsertWithTimeType.feature

```diff
@@ -57,6 +57,7 @@ Feature: Insert with time-dependent types
     Then a ExecutionError should be raised at runtime:Storage Error: The data type does not meet the requirements. Use the correct type of data.
     And drop the used space
 
+  @distonly
```

Comment:
It should support in standalone version too.

Reply:
Fixed.